scripts/deploy/databend-query-cluster-3-nodes.sh

```diff
@@ -43,7 +43,7 @@ if [ "$mode" == "boot" ]; then
 		--log-dir ./_logs1 \
 		--raft-api-port 28103 \
 		&
-	python scripts/ci/wait_tcp.py --timeout 5 --port 9191
+	python3 scripts/ci/wait_tcp.py --timeout 5 --port 9191
```

Comment:
is the `python3` command available in the major os environment like Darwin, ubuntu, fedora? q.q

Reply:
ubuntu and Darwin I think the answer `yes` 

And I saw `python3` command in `dev_setup` https://github.com/datafuselabs/databend/blob/main/scripts/setup/dev_setup.sh#L298

or maybe we can make the command from `setup_dev` too?

Reply:
agree, we can take the python commands the same among the script files

Reply:
users `env` may have both `python and python3` we can get the **python env command** in setup  ln the python3 command.